courses/util.py

```diff
@@ -65,17 +65,19 @@ def get_room(building_code, room_number):
     return room
 
 
-def add_meetings(section, meetings):
+def set_meetings(section, meetings):
+    for meeting in section.meetings.all():
+        meeting.delete()
```

Comment:
You should be able to replace this with something like `section.meetings.all().delete()`